### PR TITLE
Gok-236 | Fix. spelling in Namma clinic

### PR DESCRIPTION
**Pull request by SanoferSameera**

### CODE REVIEW

File: masterdata/configuration/liquibase/liquibase.xml [hunk 1]
```diff
@@ -487,7 +487,7 @@
             INSERT INTO inv_stockroom (name, creator, date_created, location_id, uuid)
             VALUES ('Hosakote UPHC StockRoom', 1, NOW(), @locationidHUPSC, uuid());
             INSERT INTO inv_stockroom (name, creator, date_created, location_id, uuid)
-            VALUES ('Hosakote Nama Clinic StockRoom', 1, NOW(), @locationidHNamaClinic, uuid());
```
Comment (mohan-13) [inline masterdata/configuration/liquibase/liquibase.xml:490]: Dont modify existing liquibase entries, this will cause checksum errors

Comment (mohan-13) [reply masterdata/configuration/liquibase/liquibase.xml:490]: May write a new liquibase for updates

Comment (SanoferSameera) [reply masterdata/configuration/liquibase/liquibase.xml:490]: sure @mohan-13

**Review by mohan-13 — APPROVED** [review]

File: masterdata/configuration/liquibase/liquibase.xml [hunk 2]
```diff
@@ -499,4 +499,18 @@
         </sql>
     </changeSet>
 
+    <changeSet id="GOK-236-update-inv-StockRooms"  author="Inventory">
+        <preConditions onFail="MARK_RAN">
+            <sqlCheck expectedResult="1">select count(*) from location where name = 'Hosakote Namma Clinic'</sqlCheck>
```
Comment (gokultw) [inline masterdata/configuration/liquibase/liquibase.xml:504]: <sqlCheck expectedResult="1">select count(*) from location where name = 'Hosakote Namma Clinic'</sqlCheck>

Expected condition is not correct,  it should be 'Hosakote Nama Clinic'

**Pull request merged**